src/tt_store.erl

```diff
@@ -1,9 +1,131 @@
 -module(tt_store).
+-author("Adam Szlachta").
 
--export([find_by_time/2]).
+-behaviour(gen_server).
 
--record(talk, {title :: string(), start_time :: calendar:datetime(), end_time :: calendar:datetime(), location :: string()}).
+-include_lib("stdlib/include/ms_transform.hrl").
+
+%% API
+-export([start_link/0, add/4, find_by_time/2, find_by_name/1, list/0]).
+
+%% gen_server callbacks
+-export([init/1, handle_call/3, handle_cast/2, handle_info/2, terminate/2, code_change/3]).
+
+-define(SERVER, ?MODULE).
+-define(TABLE, talks).
+
+-record(talk,  {title :: string(), start_time :: calendar:datetime(), end_time :: calendar:datetime(), location :: string()}).
```

Comment:
The record definition should be moved to a separate file: ROOT/src/talks_tweeter.hrl. The file should contain record definition and the be included in the source files.

It could be placed in ROOT/include/talks_tweeter.hrl too, but incldues in the `include/` dir are intended for clients implementation while the `#talk{}` is internal struct.